testutils/utils.go

```diff
@@ -45,41 +44,42 @@ func FreePort(t testing.TB) int {
 	return port
 }
 
-// MakeTempDir creates a directory in the current path for a test. Caller is
```

Comment:
No changes, just formatting lines better, 80 char limit is too small
The same change is made to other method comments in this file

Reply:
Oops, sorry I wasn't aware we were following the 80 char limit per line. I see that we have a mix throughout the code base, sometimes we stick to 80 chars, other times we use 120 chars. I might have been looking at too many examples with 120 chars and thought that's what we're going for.
I can revert those changes :)

Reply:
No need to revert! (more typing...)